trtools/mergeSTR/mergeSTR.py

```diff
@@ -172,19 +179,44 @@ def GetRefAllele(current_records, mergelist):
     ref : str
        Reference allele string. Set to None if conflicting references are found.
     """
-    refs = []
+
+    def DefaultKey(record: trh.TRRecord):
+        return record.vcfrecord.REF
+
+    def HipstrKey(record: trh.TRRecord):
+        return record.ref_allele
+
+    refs: List[str] = []
+    ref_picker = DefaultKey
+    if vcfType == trh.VcfTypes.hipstr:
+        ref_picker = HipstrKey
+
     chrom = ""
     pos = -1
     for i in range(len(mergelist)):
         if mergelist[i]:
-            chrom = current_records[i].CHROM
-            pos = current_records[i].POS
-            refs.append(current_records[i].REF.upper())
+            chrom = current_records[i].chrom
+            pos = current_records[i].pos
+            refs.append(ref_picker(current_records[i]).upper())
```

Comment:
When this comes from the tr harmonizer, it'll already be uppercase

Reply:
Conversion to upper case now only happens when non harmonized REF is used